common/test/acceptance/pages/lms/problem.py

```diff
@@ -147,6 +146,12 @@ def is_focus_on_problem_meta(self):
         """
         return self.q(css='.problem-header').focused
 
+    def is_focus_on_submit_notification(self):
+        """
+        Check for focus submit notification.
+        """
+        return self.q(css='.notification-response').focused
```

Comment:
Given that we intend for notification-response to ONLY be used for submits, I think it would be easier to understand if the class name made that clear ("notification-response" seems generic to me). How do you feel about renaming it to notification-submit, since the other option is notification-hint?